Gemfile

```diff
@@ -107,10 +107,10 @@ gem 'awesome_print', require: false
 
 
 group :development, :test do
-  gem 'rubocop',             '=0.75.0', require: false
-  gem 'rubocop-rails',       '=2.5.2',  require: false
-  gem 'rubocop-rspec',       '=1.35.0', require: false
-  gem 'rubocop-performance', '=1.6.1', require: false
+  gem 'rubocop',             '= 1.25.1', require: false
```

Comment:
HoundCI doesn't say that it supports this version. 
But perhaps we should try it anyway and if it _really_ doesn't support it, we can either go back a few versions or forward.